packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -402,7 +402,7 @@ const main = (opts) => {
         .then((preset) => {
             switch (preset) {
                 case HELLO_WORLD_TEST_PROJECT:
-                    return generateHelloWorld({projectId: 'hello-world'}, opts)
```

Comment:
why did you need to change this, was this never worked?

Reply:
i guess hello world doesn't matter, as long as it doesn't block the release, we are fine

Reply:
The name key was `name: projectId: 'hello-world` in the generated hello-world project package.json file. 
Thus the smoke test script was failing and failing the build.

The changes fixes the name key to `name: 'hello-world'` and builds now the smoke test script can run without failing the build.